chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
+	echo "jenkins_fold:start:Install Python requirements"
+	pip install --upgrade pip;                              \
+	pip install --upgrade setuptools;                       \
+	pip install -r requirements.txt
+	echo "jenkins_fold:end:Install Python requirements"
+
```

Comment:
if you've already got a patch , then I will leave as is and we can fix subsequently, no point implementing the same in 2 different patches, this is just a minor DRY improvement